content/news/051/index.md

```diff
@@ -73,6 +73,19 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Learning Material Updates
 
+### Xkcd Machine
+
+![XKCD Machine](xkcd_machine.gif)
+_A small part of the XKCD Machine_
+
+Chromakode released a detailed [blog post][xkcd_machine_blogpost] about their implementation of [xkcd "Machine"][xcd_machine_playable].
+
+While Machine was written in React/Typescript, and used [Rapier][xkcd_rapier], a physics engine written in Rust.
```

Comment:
ah I rewrote that sentence multiple times and in the end it doesn't make sense; for context I was hesitating between:
- While Machine was written in React/Typescript, it's using [Rapier][xkcd_rapier]: a physics engine written in Rust.
- Machine was written in React/Typescript, and used [Rapier][xkcd_rapier], a physics engine written in Rust.